src/hiero_sdk_python/file/file_id.py

```
@@ -85,18 +85,15 @@ def from_string(cls, file_id_str: str) -> 'FileId':
             shard, realm, file, checksum = parse_from_string(file_id_str)
 
-            file_id: FileId = cls(
-                shard=int(shard),
-                realm=int(realm),
-                file=int(file)
-            )
-            object.__setattr__(file_id, 'checksum', checksum)
+            file_id: FileId = cls(shard=int(shard), realm=int(realm), file=int(file))
+            object.__setattr__(file_id, "checksum", checksum)
 
             return file_id
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)